draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -315,6 +317,12 @@ We also use strings in double quotes to represent ASCII-encoded literals. For ex
 
 Those rules will be used explicitly on every operation. See also [Serialize](#serialize).
 
+### Header and Presentation Header Usage
+
+There are two special values defined by the BBS Scheme; the `header` and the `presentation_header`. The `header` value is chosen by the Signer and is bound to both a BBS signature and the BBS proofs generated using that signature. Specifically, the Prover is required to reveal the `header` to the proof Verifier, during every BBS proof presentation. As a result, the Signer SHOULD NOT include to the `header` any identifying information, that may have the potential of compromising the Prover's privacy. Suitable use cases taking advantage of the `header` value, are binding a BBS signature (and subsequent BBS proofs) to a specific application, deployment or domain, or binding the signature to specific sets of metadata etc..
+
+Similarly, the Prover can choose a `presentation_header` value to be bound to the BBS proof (in contrast to the `header` value that is chosen by the Signer and is bound to both BBS proof and signature). Verifying a BBS proof will guarantee the authenticity and integrity of the `presentation_header`. This makes it suitable for applications where the Prover may want to "sign" a message in addition to creating their BBS proof, in a way similar to how group signatures may work. Specifically, the Verifier will know that the message included in the `presentation_header` was signed by a member of the group, where group here the Provers having received valid signatures from the specific Signer, but will not know from exactly which member of the group the message was signed. Another application of the `presentation_header` is to guarantee freshness of a BBS proof and to avoid replay attacks. An example of how this could be achieved, is by including a random value supplied by the Verifier to the `presentation_header` value.
```

Comment:
Similarly, the Prover can choose a `presentation_header` value to be bound to the BBS proof (in contrast to the `header` value that is chosen by the Signer and is bound to both BBS proof and signature). Verifying a BBS proof will guarantee the authenticity and integrity of the `presentation_header`. This makes it suitable for applications where the Prover may want to "sign" a message in addition to creating their BBS proof, in a way similar to how group signatures can work. Specifically, the Verifier will know that the message included in the `presentation_header` was "signed" by a member of the group, where group here being the Provers having received valid signatures from the specific Signer, but will not know from exactly which member of the group the message was signed. Another application of the `presentation_header` is to guarantee freshness of a BBS proof and to avoid replay attacks. An example of how this could be achieved, is by including a random value supplied by the Verifier to the `presentation_header` value.

Reply:
Just some small English fixes in the above.